### PR TITLE
Add the ability to dump EnC delta PDB to pdb2xml tool

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
@amcasey

File: src/Tools/Source/Pdb2Xml/PdbToXmlApp.cs [hunk 1]
```diff
@@ -80,4 +103,12 @@ public static void GenXmlFromPdb(string exePath, string pdbPath, string outPath,
             }
         }
     }
+
+    public static void GenXmlFromDeltaPdb(string pdbPath, string outPath, PdbToXmlOptions options)
+    {
+        using (var deltaPdb = new FileStream(pdbPath, FileMode.Open, FileAccess.Read))
+        {
+            File.WriteAllText(outPath, PdbToXmlConverter.DeltaPdbToXml(deltaPdb, Enumerable.Range(0x06000001, 0xff)));
```
Comment (amcasey) [inline src/Tools/Source/Pdb2Xml/PdbToXmlApp.cs]: Magic constants might benefit from comments. :)

**Comment by amcasey** [comment]
:+1:

**Comment by KevinH-MS** [comment]
LGTM (with @amcasey's comment)

**Pull request merged**